packages/app/src/components/pages/Home/useSort.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useMemo, useContext } from 'react'
+import { NominationsDataContext } from '../../../utils/context/NominationsContext';
+
+const useSort = () => {
+
+  const [sortConfig, setSortConfig] = useState({key: 'dateReceived', direction: 'ascending'})
+  const [NominationsData, setNominationsData] = useContext(NominationsDataContext)
+
+  const convertDate = (str) => {
+    const dateArr = str.split('/')
+    const year = dateArr[2]
+    const month = dateArr[0].length === 1 ? `0${dateArr[0]}` : dateArr[0]
+    const day = dateArr[1].length === 1 ? `0${dateArr[1]}` : dateArr[1]
+
+    return year + month + day
+  }
```

Comment:
This seems pretty brittle. Do we know dates will always be formatted `MM/DD/YYYY`?